SPECS/qemu/qemu.spec

```diff
@@ -217,7 +217,7 @@ Obsoletes: %{name}-system-unicore32-core <= %{version}-%{release}
 Summary:        QEMU is a FAST! processor emulator
 Name:           qemu
 Version:        6.2.0
-Release:        8%{?dist}
+Release:        9%{?dist}
```

Comment:
It seems we might have missed to update the `%changelog` :)
Every modification to a spec warrants a changelog :)

Reply:
Thanks :)